src/Microsoft.AspNet.Mvc.Core/Formatters/AcceptHeaderParser.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNet.Mvc.Formatters
+{
+    public static class AcceptHeaderParser
```

Comment:
There's not really any code sharing between this and the media type parser... should there be?